keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/README.md

```diff
@@ -327,6 +340,8 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+The operator can query pods with `pod.spec.topologySpreadConstraints.minDomains` field set.
```

Comment:
I wonder if we could extend https://github.com/kubernetes/kubernetes/issues/107556 to produce a metric.

But I wouldn't block beta graduation to this.

Reply:
We definitely need https://github.com/kubernetes/kubernetes/issues/107556... 
I have not had enough time to work on this, but will make it a priority...

> to produce a metric.

So, what metrics do you imagine?

Reply:
can you create an issue so we can discuss there?

But it would be something about which filter plugins influenced a pod scheduling decision. One counter increment for each plugin.

Reply:
Opened: https://github.com/kubernetes/kubernetes/issues/110643